docker-compose.yml

```
@@ -10,7 +10,7 @@ services:
     stdin_open: true
     working_dir: /home/package
     environment:
-      - MEILISEARCH_HOST=http://meilisearch:7700
+      - MEILISEARCH_URL=http://meilisearch:7700
```

Can you change to MEILISEARCH_HOST again?
```suggestion
      - MEILISEARCH_HOST=http://meilisearch:7700
```

All the other SDKs, share the same principles, and if I change here I will have to change in every other repo, I agree with your suggestion and I will create an issue for that :)

But for now to merge your PR I have to ask you to change the docker config back ;)